### PR TITLE
add pascal seitz

### PR DESCRIPTION
**Pull request by PSeitz**

### CODE REVIEW

**Comment by jamesmunns** [comment]
@PSeitz could you please move your name alphabetically? Thanks!

File: README.md [hunk 1]
```diff
@@ -77,6 +77,9 @@ If you have suggestions for where this list should live (other than @jamesmunns'
     * [YouTube Channel](https://www.youtube.com/channel/UCpeX4D-ArTrsqvhLapAHprQ), [Twitch Channel](https://www.twitch.tv/ryanlevick)
     * [GitHub](https://github.com/rylev), [Twitter](https://twitter.com/ryan_levick)
     * Teaching, Web Assembly
+* Pascal Seitz
+    * [Twitch Channel](https://www.twitch.tv/Paskarlo), [GitHub](https://github.com/pseitz)
```
Comment (jamesmunns) [inline README.md]:
```suggestion
    * [Twitch Channel](https://www.twitch.tv/Paskarlo)
    * [GitHub](https://github.com/pseitz)
```

**Comment by PSeitz** [comment]
@jamesmunns done, previously I inserted by surname

**Comment by jamesmunns** [comment]
Thanks!

**Pull request merged**